### PR TITLE
feat(breadcrumb): create breadcrumb component

### PR DESCRIPTION
**Pull request by marinasmah**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/breadcrumbs/Breadcrumbs.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { ifDefined } from "lit-html/directives/if-defined";
+
+type NavCrumb = {
+  link: string;
+  label: string;
+};
+
+@customElement("md-breadcrumbs")
+export class Breadcrumbs extends LitElement {
+  @property({ type: String }) label = "";
```
Comment (futurethang) [inline web-components/src/components/breadcrumbs/Breadcrumbs.ts:14]: according to the wai-aria best practice this should default to "Breadcrumb"

**Review by futurethang — CHANGES_REQUESTED** [review]

File: web-components/src/components/breadcrumbs/Breadcrumbs.ts (hunk 1 above)
Comment (marinasmah) [reply web-components/src/components/breadcrumbs/Breadcrumbs.ts]: fixed

Comment (futurethang) [reply web-components/src/components/breadcrumbs/Breadcrumbs.ts]: Did it get changed back? I still see an empty string.

Comment (marinasmah) [reply web-components/src/components/breadcrumbs/Breadcrumbs.ts]: Done

**Review by yananym — APPROVED** [review]

**Pull request merged**